packages/backend/src/config/parseConfig.ts

```diff
@@ -53,10 +53,16 @@ export type LightdashConfigIn = {
 
 export type LightdashConfig = {
     version: '1.0';
+    rudder: RudderConfig;
     mode: LightdashMode;
     projects: Array<DbtProjectConfig>;
 };
 
+export type RudderConfig = {
+    writeKey: string | null;
+    dataPlaneUrl: string | null;
```

Comment:
Since we have a fallback value I don't think we should have the `null` type here.